qml/quicksettings/QuickSettings.qml

```diff
@@ -63,13 +66,13 @@ Item {
     }
 
     QuickSettingsToggle {
-        id: lockedToggle
+        id: soundToggle
```

Comment:
These changes should not be part of this PR.

We should discuss in a separate issue. Obviously it would be nice to have a quick button to mute all sounds, but not every watch has a speaker rendering this button entirely useless.
It also replaces an already existing function, which might not be desirable right now.

Reply:
removed that commit